CODEOWNERS

```diff
@@ -1 +1 @@
-# release configuration
+*   @hashicorp/vault-crypto
```

Comment:
Working with Luciano on getting the vault-crypto group write access to the repo.